app/services/users.js

```diff
@@ -0,0 +1,23 @@
+const { User } = require('../models/'),
+  errors = require('../errors');
+
+exports.getUserByEmail = email => {
+  return User.findOne({
```

Comment:
if you are using functions inside the model justput all this logic in the model and delete the service

Reply:
I did this to have one more layer of abstraction. Sometimes you need to do something before the request to the database. I wanted to maintain that issue clean.
Also, you told me before to use services when a request to the database is needed.